introduction_to_amazon_algorithms/forecasting_services_comparison/forecast_example.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<p>How long should this job take? Writing a <code>while True</code> is risky in that if something goes differently than expected, you'll be stuck in an infinite loop. Can you do something like <code>while tts_status == "CREATING"</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>